M29 New composer documentation (#235)
exoplatform/exo-documentation: docs/GettingStarted.rst

Posting in the activity stream via the New Composer
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~	

> A new way to share in the activity stream is available via a clickable text "Post something to your network" from home page and "Post something to 'Space Name' " from space's home page.

Reviewer: ```suggestion
A new way to share in the activity stream is available via a clickable text "Post something to your network" from the home page and "Post something to '$Space Name' " from space's home page.
```

Home page composer: `/Public/Activity Stream Documents` under the `Personal Documents` drive.
Space composer : `/Activity Stream Documents` under the space’s drive.

You can create a new folder from the select folder drawer by clicking on the appropriate icon:

|image239|

> You only have to set the name of the automatically created folder, then enter to register the action. 

Reviewer: ```suggestion
You just have to set the name for the automatically created folder, and then press enter to validate the action. 
```
